cli/cmd/root.go

```diff
@@ -17,10 +18,11 @@ import (
 )
 
 const (
-	defaultNamespace      = "linkerd"
-	defaultCNINamespace   = "linkerd-cni"
-	defaultClusterDomain  = "cluster.local"
-	defaultDockerRegistry = "gcr.io/linkerd-io"
+	defaultNamespace        = "linkerd"
+	defaultCNINamespace     = "linkerd-cni"
+	defaultClusterDomain    = "cluster.local"
+	defaultDockerRegistry   = "gcr.io/linkerd-io"
+	defaultContextNamespace = "default"
```

Comment:
you can use `corev1.NamespaceDefaul` .

Reply:
Since the constant is used in only one space, I have replaced it with `corev1.NamespaceDefault` in the function return. Think this might show intent a bit better than the solution I had initially.